orca-kayenta/src/main/kotlin/com/netflix/spinnaker/orca/kayenta/pipeline/RunCanaryPipelineStage.kt

```diff
@@ -37,6 +40,13 @@ class RunCanaryPipelineStage(
   private val log = LoggerFactory.getLogger(javaClass)
 
   override fun taskGraph(stage: Stage, builder: TaskNode.Builder) {
+      val context = stage.mapTo<RunCanaryContext>()
```

Comment:
the one caveat we talked about here.. is that we will query the canary config every run which is suboptimal, but can be tracked separately.

Reply:
I was going to open an issue against Spinnaker/orca, but I can't find an `issues` tab for it. What's up with that?

Reply:
https://github.com/spinnaker/spinnaker/issues/5006